protobuf-provider/src/main/java/io/confluent/kafka/schemaregistry/protobuf/ProtobufSchema.java

```diff
@@ -566,10 +559,10 @@ protected DynamicSchema toDynamicSchema() {
   private static DynamicSchema toDynamicSchema(
       String name, ProtoFileElement rootElem, Map<String, ProtoFileElement> dependencies
   ) {
-    log.trace("*** toDynamicSchema: {}", rootElem.toSchema());
+    log.trace("*** toDynamicSchema: {}", ProtobufSchemaUtils.toString(rootElem));
```

Comment:
Can we guard this with an if-statement, to avoid evaluating the expression if trace logs are disabled ?

Reply:
Good catch, I'll fix